.github/workflows/tests-cli.yml

```diff
@@ -103,9 +108,11 @@ jobs:
         run: make test
 
   push-to-tinybird:
-    if: always() && github.ref == 'refs/heads/master'
+    if: always() && github.ref == 'refs/heads/master' && github.repository == 'localstack/localstack'
```

Comment:
A fork does not have the secret necessary to push to Tinybird...

Reply:
Yes that was my reasoning for adding this change. This would prevent the job from failing is it will not run at all.